datasets/ONS-UK-trade-in-services-by-business-characteristics/info.json

```diff
@@ -22,20 +22,24 @@
                 "value": "http://reference.data.gov.uk/id/{+period}"
             },
             "Business size": {
-                "parent": "http://gss-data.org.uk/def/family-trade/concept-scheme/employment-size-bands",
-                "value": "http://gss-data.org.uk/def/family-trade/concept-scheme/employment-size-bands/{business-size}"
+                "parent": "http://gss-data.org.uk/def/trade/property/dimension/employment-size-bands",
+                "value": "http://gss-data.org.uk/def/trade/property/dimension/employment-size-bands/{business-size}"
```

Comment:
Values should reference concepts - so for each `value` key, use `/concept-scheme/` instead of `/property/dimension/`.